### PR TITLE
Initial Setup for Devs koenj

### PR DESCRIPTION
**Pull request by kjacoby3**
This PR helps set up the initial file structure of the Web Application. I have a description of the folders in the README.
The server should also be an easy spin-up if you have a proper .env file.

### CODE REVIEW

**Review by Brycew73 — APPROVED** [review]
Looks good.

**Review by regreaves — APPROVED** [review]
:)

File: README.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
 # YCPHacks2023
```
Comment (Trey-Mac) [inline README.md:2]: this is programming

File: apps/api/src/index.mjs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+registerPlugins(fastify);
+
+fastify.register(hardwareCheckout);
+
+const start = async () => {
+    try {
+        await fastify.listen(process.env.PORT);
+    } catch (err) {
+        fastify.log.error(err);
+        process.exit(1);
```
Comment (regreaves) [inline apps/api/src/index.mjs]: process.exitCode = 1;

File: apps/web/src/index.mjs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import hardwareCheckout from "@ycphacks/hardware-checkout-web";
+import core from "@ycphacks/core-web"
+
+// Creates a new Fastify instance
+const fastify = Fastify({ logger: true });
+
+// Register each sub-app as a plugin
+fastify.register(eventControlPanel);
+fastify.register(hardwareCheckout);
+fastify.register(core);
```
Comment (regreaves) [inline apps/web/src/index.mjs]: Should core be registered first?

File: apps/web/src/index.mjs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+fastify.setNotFoundHandler(async function (request, reply) {
+    return reply.viewCore('404.pug');
+});
+const start = async () => {
+    try {
+        await fastify.listen(process.env.PORT);
+    } catch (err) {
+        fastify.log.error(err);
+        process.exit(1);
```
Comment (regreaves) [inline apps/web/src/index.mjs]: process.exitCode = 1;

**Review by regreaves — CHANGES_REQUESTED** [review]
:)

**Review by regreaves — APPROVED** [review]
🦅

**Review by Trey-Mac — APPROVED** [review]
this makes sense to me

**Pull request merged**